src/screens/ContactScreen/index.js

```diff
@@ -383,7 +446,7 @@ const ContactScreen = ({navigation}) => {
         </View>
       )}
 
-      {isRecyclerViewShown && !isSearchMode && (
+      {isRecyclerViewShown && usersSearch.length <= 0 && !isLoadingSearchResult && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for showing the regular user list has been updated to also check if there are no search results and if the search is not loading. This is a good change as it ensures that the regular user list is only shown when appropriate, improving the user experience.



<!-- This is an auto-generated comment by OSS CodeRabbit -->